xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -220,7 +218,6 @@ struct UserProfile: Equatable, Codable, Hashable {
     let pfp: ProfilePicVariant
     let bio: UserProfileBio?
     let category: UserCategory
-    let resolutionStatus: ResolutionStatus
```

Comment:
Dropped this field because I felt we were mixing concerns.

`UserProfile`s contain a user's "display" metadata.
`AddressBookEntry`s contain noosphere-level user data like name resolution status.